pkg/cloudprovider/aws/suite_test.go

```diff
@@ -51,8 +52,27 @@ var launchTemplateCache *cache.Cache
 var fakeEC2API *fake.EC2API
 var controller reconcile.Reconciler
 
+type singletonFS struct {
+	filename string
+	contents []byte
+}
+
+func (s *singletonFS) Open(name string) (fs.File, error) {
+	panic("not implemented; only ReadFile needed")
+}
+
+func (s *singletonFS) ReadFile(name string) ([]byte, error) {
+	if name == s.filename {
+		return s.contents, nil
+	}
+	return nil, fs.ErrNotExist
+}
+
 func TestAPIs(t *testing.T) {
-	ctx = TestContextWithLogger(t)
+	ctx = filesys.Inject(TestContextWithLogger(t), &singletonFS{
```

Comment:
Thoughts on just injecting this for the test where you need it?

Reply:
See my other response.